src/features/favorites/pages/FavoritesWrapper.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useContext, useReducer } from 'react'
+
+import {
+  Action,
+  initialFavoritesState,
+  favoritesReducer,
+  FavoritesState,
+} from 'features/favorites/pages/reducer'
+
+export interface IFavoritesContext {
+  favoritesState: FavoritesState
+  dispatch: React.Dispatch<Action>
+}
+
+export const FavoritesContext = React.createContext<IFavoritesContext | null>(null)
+
+export const FavoritesWrapper = ({ children }: { children: Element }) => {
+  const [favoritesState, dispatch] = useReducer(favoritesReducer, initialFavoritesState)
+
+  return (
+    <FavoritesContext.Provider value={{ favoritesState, dispatch }}>
```

Comment:
C'est une bad practice de redéclarer un objet comme valeur d'un Provider de Context.
Il faut normalement s'arranger pour que la valeur soit celle de l'état / d'un état du composant Provider.
Cf. `SplashScreenProvider`.
```suggestion
    <FavoritesContext.Provider value={contextValue}>
```

Reply:
OK

Reply:
Fixed